ufl/finiteelement/elementlist.py

```diff
@@ -145,6 +144,10 @@ def show_elements():
 register_element("HDiv Trace", "HDivT", 0, L2, "identity", (0, None), any_cell)
 register_element("Hellan-Herrmann-Johnson", "HHJ", 2, HDivDiv,
                  "double contravariant Piola", (0, None), ("triangle",))
+register_element("Nonconforming Arnold-Winther", "AWnc", 2, HDivDiv,
+                 "double contravariant Piola", (0, None), ("triangle", "tetrahedron"))
+register_element("Conforming Arnold-Winther", "AWc", 2, HDivDiv,
+                 "double contravariant Piola", (0, None), ("triangle", "tetrahedron"))
```

Comment:
FIAT code says that degree must be 3 for conforming, and 2 for nonconforming AW elements.  Here it is 0 to infinity.  That is inconsistent.

Reply:
The conforming element is defined for orders >= 3 (depending on how you count...).  The nonconforming element is only defined based on (constrained) quadratics.  

The FIAT implementation for the conforming case is currently only degree 3, but this could be generalized at some point if desired.  So I will change the nonconforming case to (2, 2) and the conforming for (3, None) so we won't have to change UFL if FIAT gets an upgrade here.